app/routes/rental_routes.py

```diff
@@ -0,0 +1,79 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.rental import Rental
+from app.models.customer import Customer
+from app.models.video import Video
+from datetime import timedelta, datetime
+from app.models.helper import invalid_rental_data
+
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def post_rental():
+    request_body = request.get_json()
+    invalid_request = invalid_rental_data(request_body)
+    if invalid_request:
+        return jsonify(invalid_request), 400
+
+    one_video = Video.query.get(request_body["video_id"])
+    one_customer = Customer.query.get(request_body["customer_id"])
+    if one_video is None:
+        return jsonify({"Bad Request": f"Video not found."}), 404
+    elif one_customer is None:
+        return jsonify({"Bad Request": f"Customer not found."}), 404
+    video_current_checked_out = Rental.query.filter_by(checked_out=True, video_id=one_video.id).count()
+    available_inventory = one_video.total_inventory - video_current_checked_out
```

Comment:
Consider moving the logic for calculating the checked out count and available inventory to helper methods in the Video or Rental model.